components/local-app/main.go

```diff
@@ -92,11 +97,39 @@ func run(host, sshConfig string) error {
 		Context: context.Background(),
 		Token:   tkn,
 		Log:     logrus.NewEntry(logrus.New()),
+		ReconnecitonHandler: func() {
+			if b != nil {
+				b.FullUpdate()
+			}
+		},
 	})
 	if err != nil {
 		return err
 	}
-	b := bastion.New(client, cb)
+
+	b = bastion.New(client, cb)
+	go http.ListenAndServe("localhost:5000", http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
```

Comment:
it's just temporary, it will be removed in future PRs and replaced by proper grpc-web interface